images/cpg_hail_gcloud/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_hail:0.2.134.cpg2-1
 
-ARG VERSION="0.2.134.cpg2"
```

Comment:
Argh… I didn't think to strip quote marks in the latest edition of that workflow regex…